{"expand":"renderedFields,names,schema,operations,editmeta,changelog,versionedRepresentations","id":"22078","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/22078","key":"EUCA-6665","fields":{"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803},"timespent":null,"customfield_13100":null,"project":{"self":"https://eucalyptus.atlassian.net/rest/api/2/project/10000","id":"10000","key":"EUCA","name":"Eucalyptus","avatarUrls":{"48x48":"https://eucalyptus.atlassian.net/secure/projectavatar?pid=10000&avatarId=10011","24x24":"https://eucalyptus.atlassian.net/secure/projectavatar?size=small&pid=10000&avatarId=10011","16x16":"https://eucalyptus.atlassian.net/secure/projectavatar?size=xsmall&pid=10000&avatarId=10011","32x32":"https://eucalyptus.atlassian.net/secure/projectavatar?size=medium&pid=10000&avatarId=10011"}},"customfield_11000":null,"customfield_11001":null,"fixVersions":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/12105","id":"12105","description":"Eucalyptus 3.4.0","name":"3.4.0","archived":false,"released":true,"releaseDate":"2013-10-24"}],"aggregatetimespent":null,"resolution":{"self":"https://eucalyptus.atlassian.net/rest/api/2/resolution/1","id":"1","description":"A fix for this issue is checked into the tree and tested.","name":"Fixed"},"customfield_10310":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10209","value":"CentOS 6","id":"10209"}],"customfield_10311":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10306","value":"System","id":"10306"}],"customfield_11401":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=tony","name":"tony","key":"tony","accountId":"557058:353956db-b78c-4f3b-9096-f8f58e6709c8","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue"},"displayName":"Tony Beckham","active":true,"timeZone":"America/Tijuana"},"customfield_11400":null,"customfield_10500":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10403","value":"Not Applicable","id":"10403"},"customfield_10302":null,"customfield_10104":["com.atlassian.greenhopper.service.sprint.Sprint@4ea50324[id=26,rapidViewId=64,state=CLOSED,name=3.4 Sprint 4,goal=<null>,startDate=2013-08-27T04:24:27.505Z,endDate=2013-09-17T04:24:00.000Z,completeDate=2013-09-17T00:38:04.117Z,sequence=26]","com.atlassian.greenhopper.service.sprint.Sprint@2a9f133b[id=27,rapidViewId=34,state=CLOSED,name=3.4 Sprint 5,goal=<null>,startDate=2013-09-17T19:21:02.124Z,endDate=2013-10-04T22:21:00.000Z,completeDate=2013-10-09T15:31:36.581Z,sequence=27]"],"customfield_10303":null,"customfield_10105":null,"customfield_10700":null,"customfield_10304":null,"customfield_10701":null,"customfield_10306":{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10201","value":"Medium","id":"10201"},"customfield_10702":null,"customfield_10703":null,"resolutiondate":"2013-09-23T20:00:11.141-0500","customfield_10704":null,"customfield_10308":null,"customfield_10705":null,"customfield_10309":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/customFieldOption/10204","value":"KVM","id":"10204"}],"workratio":-1,"watches":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/EUCA-6665/watchers","watchCount":8,"isWatching":false},"lastViewed":null,"created":"2013-06-25T01:24:09.684-0500","customfield_12000":null,"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/2","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/critical.svg","name":"Critical","id":"2"},"customfield_12400":null,"customfield_12201":null,"customfield_12600":"{repository={count=2, dataType=repository}, json={\"cachedValue\":{\"errors\":[],\"summary\":{\"repository\":{\"overall\":{\"count\":2,\"lastUpdated\":\"2013-09-12T19:13:55.000-0500\",\"dataType\":\"repository\"},\"byInstanceType\":{\"github\":{\"count\":2,\"name\":\"GitHub\"}}}}},\"isStale\":true}}","customfield_10300":null,"customfield_10102":null,"labels":[],"customfield_10301":null,"customfield_11700":null,"timeestimate":null,"aggregatetimeoriginalestimate":null,"versions":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/version/11305","id":"11305","name":"3.2.2","archived":false,"released":true,"releaseDate":"2013-04-01"}],"customfield_11901":null,"issuelinks":[{"id":"20018","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLink/20018","type":{"id":"10102","name":"Duplicate","inward":"is duplicated by","outward":"duplicates","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLinkType/10102"},"inwardIssue":{"id":"22858","key":"EUCA-7071","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/22858","fields":{"summary":"new instance creation fails with \"Could not execute JDBC batch update\"","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/2","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/critical.svg","name":"Critical","id":"2"},"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803}}}},{"id":"19254","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLink/19254","type":{"id":"10103","name":"Relates","inward":"relates to","outward":"relates to","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLinkType/10103"},"outwardIssue":{"id":"17250","key":"EUCA-4211","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/17250","fields":{"summary":"Private network index allocation is not thread safe","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/4","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/minor.svg","name":"Minor","id":"4"},"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803}}}},{"id":"19253","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLink/19253","type":{"id":"10103","name":"Relates","inward":"relates to","outward":"relates to","self":"https://eucalyptus.atlassian.net/rest/api/2/issueLinkType/10103"},"inwardIssue":{"id":"22364","key":"EUCA-6882","self":"https://eucalyptus.atlassian.net/rest/api/2/issue/22364","fields":{"summary":"when NC server reboot,unable start bfebs instance -\" Could not execute JDBC batch update\"","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"priority":{"self":"https://eucalyptus.atlassian.net/rest/api/2/priority/3","iconUrl":"https://eucalyptus.atlassian.net/images/icons/priorities/major.svg","name":"Major","id":"3"},"issuetype":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issuetype/1","id":"1","description":"A problem which impairs or prevents the functions of the product.","iconUrl":"https://eucalyptus.atlassian.net/secure/viewavatar?size=xsmall&avatarId=14803&avatarType=issuetype","name":"Bug","subtask":false,"avatarId":14803}}}}],"assignee":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=tony","name":"tony","key":"tony","accountId":"557058:353956db-b78c-4f3b-9096-f8f58e6709c8","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue"},"displayName":"Tony Beckham","active":true,"timeZone":"America/Tijuana"},"updated":"2014-06-16T16:13:06.221-0500","status":{"self":"https://eucalyptus.atlassian.net/rest/api/2/status/6","description":"The issue is considered finished, the resolution is correct. Issues which are closed can be reopened.","iconUrl":"https://eucalyptus.atlassian.net/images/icons/statuses/generic.png","name":"Closed","id":"6","statusCategory":{"self":"https://eucalyptus.atlassian.net/rest/api/2/statuscategory/3","id":3,"key":"done","colorName":"green","name":"Done"}},"components":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/component/10002","id":"10002","name":"Cloud Controller"}],"timeoriginalestimate":null,"description":"I have patch as EUCA-4211, but this issue can also appear, as the comment of [~sjones]  \r\n\r\n\"Concurrent private network index allocation is now supported, we do not support concurrent allocation for the extant network. This means you may see failures in tests, but the cause will be insertion into 'metadata_extant_network' rather than 'metadata_network_indices'\"\r\n\r\nThis issue have not fixed. Please help to check.\r\n\r\n{code}\r\n2013-06-24 16:36:55 TRACE 000000844 Entities                 | javax.persistence.RollbackException: Error while committing the transaction\r\njavax.persistence.RollbackException: Error while committing the transaction\r\n         at org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:93)\r\n         at com.eucalyptus.entities.Entities$CascadingTx$TxState.commit(Entities.java:846)\r\n         at com.eucalyptus.entities.Entities$CascadingTx.commit(Entities.java:704)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator.<init>(ClusterAllocator.java:188)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator.<init>(ClusterAllocator.java:125)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator$SubmitAllocation$1.call(ClusterAllocator.java:159)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator$SubmitAllocation$1.call(ClusterAllocator.java:155)\r\n         at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)\r\n         at java.util.concurrent.FutureTask.run(FutureTask.java:166)\r\n         at com.eucalyptus.system.Threads$Queue.run(Threads.java:682)\r\n         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)\r\n         at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)\r\n         at java.util.concurrent.FutureTask.run(FutureTask.java:166)\r\n         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)\r\n         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)\r\n         at java.lang.Thread.run(Thread.java:679)\r\nCaused by: javax.persistence.PersistenceException: org.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n         at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1235)\r\n         at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1168)\r\n         at org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:81)\r\n         ... 15 more\r\nCaused by: org.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n         at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:96)\r\n         at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)\r\n         at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:275)\r\n         at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:262)\r\n         at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:178)\r\n         at org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:321)\r\n         at org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:51)\r\n         at org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1206)\r\n         at org.hibernate.impl.SessionImpl.managedFlush(SessionImpl.java:375)\r\n         at org.hibernate.transaction.JDBCTransaction.commit(JDBCTransaction.java:137)\r\n         at org.hibernate.ejb.TransactionImpl.commit(TransactionImpl.java:76)\r\n         ... 15 more\r\nCaused by: java.sql.BatchUpdateException: Batch entry 0 insert into metadata_extant_network (creation_timestamp, last_update_timestamp, metadata_perm_uuid, version, metadata_display_name, metadata_last_state, metadata_state, metadata_state_change_stack, metadata_account_name, metadata_account_id, metadata_unique_name, metadata_user_id, metadata_user_name, networkGroup_id, metadata_extant_network_tag, id) values ('2013-06-24 16:36:55.449000 +08:00:00', '2013-06-24 16:36:55.449000 +08:00:00', '5fe8aa6b-61cf-46de-be34-f6592e097724', '0', 'default', 'PENDING', 'PENDING', 'com.eucalyptus.system.Threads.currentStackRange(Threads.java:577)         \r\ncom.eucalyptus.system.Threads.currentStackString(Threads.java:585)  \r\ncom.eucalyptus.entities.AbstractStatefulPersistent.setState(AbstractStatefulPersistent.java:114)         \r\ncom.eucalyptus.network.ExtantNetwork.<init>(ExtantNetwork.java:138)  \r\ncom.eucalyptus.network.ExtantNetwork.bogus(ExtantNetwork.java:166)          \r\ncom.eucalyptus.network.NetworkGroup.extantNetwork(NetworkGroup.java:278)     \r\ncom.eucalyptus.cloud.run.Allocations$Allocation.getExtantNetwork(Allocations.java:219)       \r\ncom.eucalyptus.cloud.run.ClusterAllocator.setupNetworkMessages(ClusterAllocator.java:275)         \r\ncom.eucalyptus.cloud.run.ClusterAllocator.<init>(ClusterAllocator.java:187)   \r\ncom.eucalyptus.cloud.run.ClusterAllocator.<init>(ClusterAllocator.java:125)   \r\ncom.eucalyptus.cloud.run.ClusterAllocator$SubmitAllocation$1.call(ClusterAllocator.java:159)         \r\ncom.eucalyptus.cloud.run.ClusterAllocator$SubmitAllocation$1.call(ClusterAllocator.java:155)         \r\njava.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)    \r\njava.util.concurrent.FutureTask.run(FutureTask.java:166)       \r\ncom.eucalyptus.system.Threads$Queue.run(Threads.java:682)     \r\njava.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)       \r\njava.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)    \r\njava.util.concurrent.FutureTask.run(FutureTask.java:166)       \r\njava.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)      \r\njava.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)', NULL, '386196274624', '386196274624:default', '1KLTRDMNQ0OVYRRDWGTZR', 'admin', '000000003f26d785013f272eb979006c', '-2', '000000003f7544b4013f75542219000c') was aborted.  Call getNextException to see the cause.\r\n         at org.postgresql.jdbc2.AbstractJdbc2Statement$BatchResultHandler.handleError(AbstractJdbc2Statement.java:2621)\r\n         at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1837)\r\n         at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:407)\r\n         at org.postgresql.jdbc2.AbstractJdbc2Statement.executeBatch(AbstractJdbc2Statement.java:2754)\r\n         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n         at java.lang.reflect.Method.invoke(Method.java:616)\r\n         at net.sf.hajdbc.util.reflect.Methods.invoke(Methods.java:52)\r\n         at net.sf.hajdbc.sql.AbstractInvocationHandler$SimpleInvoker.invoke(AbstractInvocationHandler.java:547)\r\n         at net.sf.hajdbc.sql.DatabaseWriteInvocationStrategy$1.call(DatabaseWriteInvocationStrategy.java:92)\r\n         at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)\r\n         at java.util.concurrent.FutureTask.run(FutureTask.java:166)\r\n         at net.sf.hajdbc.util.concurrent.SynchronousExecutor.execute(SynchronousExecutor.java:90)\r\n         at java.util.concurrent.AbstractExecutorService.submit(AbstractExecutorService.java:120)\r\n         at net.sf.hajdbc.sql.DatabaseWriteInvocationStrategy.invokeAll(DatabaseWriteInvocationStrategy.java:96)\r\n         at net.sf.hajdbc.sql.DatabaseWriteInvocationStrategy.invoke(DatabaseWriteInvocationStrategy.java:62)\r\n         at net.sf.hajdbc.sql.LockingInvocationStrategy.invoke(LockingInvocationStrategy.java:61)\r\n         at net.sf.hajdbc.sql.LocalTransactionContext$1.invoke(LocalTransactionContext.java:69)\r\n         at net.sf.hajdbc.sql.AbstractInvocationHandler.invoke(AbstractInvocationHandler.java:97)\r\n         at $Proxy110.executeBatch(Unknown Source)\r\n         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)\r\n         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n         at java.lang.reflect.Method.invoke(Method.java:616)\r\n         at org.logicalcobwebs.proxool.ProxyStatement.invoke(ProxyStatement.java:100)\r\n         at org.logicalcobwebs.proxool.ProxyStatement.intercept(ProxyStatement.java:57)\r\n         at $java.io.Serializable$$EnhancerByProxool$$410c0486.executeBatch(<generated>)\r\n         at org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:70)\r\n         at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:268)\r\n         ... 23 more\r\n2013-06-24 16:36:55 ERROR 000000844 ClusterAllocator         | org.hibernate.exception.ConstraintViolationException: Could not execute JDBC batch update\r\n2013-06-24 16:36:55 ERROR 000000844 Allocations              | Aborting resource token: ResourceToken:i-77E341F5:\r\njava.lang.RuntimeException\r\n         at com.eucalyptus.cloud.run.Allocations$Allocation.abort(Allocations.java:245)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator.<init>(ClusterAllocator.java:193)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator.<init>(ClusterAllocator.java:125)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator$SubmitAllocation$1.call(ClusterAllocator.java:159)\r\n         at com.eucalyptus.cloud.run.ClusterAllocator$SubmitAllocation$1.call(ClusterAllocator.java:155)\r\n         at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)\r\n         at java.util.concurrent.FutureTask.run(FutureTask.java:166)\r\n         at com.eucalyptus.system.Threads$Queue.run(Threads.java:682)\r\n         at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)\r\n         at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)\r\n         at java.util.concurrent.FutureTask.run(FutureTask.java:166)\r\n         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1110)\r\n         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:603)\r\n         at java.lang.Thread.run(Thread.java:679)\r\n{code}\r\n","customfield_13000":null,"customfield_13200":null,"customfield_11300":null,"customfield_11500":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"timetracking":{},"customfield_10005":null,"customfield_10401":null,"customfield_10600":null,"customfield_12900":null,"attachment":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/attachment/15720","id":"15720","filename":"cloud-output.log.1","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=QinWei","name":"QinWei","key":"qinwei","accountId":"557058:37f974f7-a218-48c8-8028-3ca5698c4a9b","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"QinWei","active":true,"timeZone":"US/Central"},"created":"2013-06-25T01:24:09.706-0500","size":5432586,"mimeType":"application/octet-stream","content":"https://eucalyptus.atlassian.net/secure/attachment/15720/cloud-output.log.1"}],"aggregatetimeestimate":null,"summary":"System Mode run instance frequently error","creator":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=QinWei","name":"QinWei","key":"qinwei","accountId":"557058:37f974f7-a218-48c8-8028-3ca5698c4a9b","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"QinWei","active":true,"timeZone":"US/Central"},"subtasks":[],"reporter":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=QinWei","name":"QinWei","key":"qinwei","accountId":"557058:37f974f7-a218-48c8-8028-3ca5698c4a9b","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FavatarId%3D10122%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26avatarId%3D10122%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26avatarId%3D10122%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/48ab31420165afbcc95b0ca36cc80b29?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26avatarId%3D10122%26noRedirect%3Dtrue"},"displayName":"QinWei","active":true,"timeZone":"US/Central"},"aggregateprogress":{"progress":0,"total":0},"customfield_12100":"0|i00kd3:","customfield_10000":"3_*:*_1_*:*_4807693_*|*_10001_*:*_1_*:*_6658656796_*|*_10000_*:*_1_*:*_226505630_*|*_10006_*:*_1_*:*_0_*|*_10003_*:*_1_*:*_952991369","customfield_10001":"2013-06-27T16:19:15.281-0500","customfield_10002":null,"customfield_10200":null,"customfield_10003":null,"customfield_12500":null,"customfield_10400":null,"customfield_10116":null,"customfield_11600":["kedwards(kedwards)","QinWei(qinwei)","sjones(sjones)","tony(tony)"],"customfield_10117":null,"environment":null,"customfield_11800":null,"customfield_11802":null,"customfield_11805":null,"duedate":null,"progress":{"progress":0,"total":0},"customfield_11806":null,"votes":{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/EUCA-6665/votes","votes":0,"hasVoted":false},"comment":{"comments":[{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/22078/comment/49029","id":"49029","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=kedwards","name":"kedwards","key":"kedwards","accountId":"557058:f9c2f8da-1fc6-4aa7-8fb0-750e7e422b6b","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue"},"displayName":"Ken Edwards","active":true,"timeZone":"America/Los_Angeles"},"body":"Need furthering information to reproduce the problem. One possible use case, run two instances with in the same security group may cause this problem with conflicting security groups.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=kedwards","name":"kedwards","key":"kedwards","accountId":"557058:f9c2f8da-1fc6-4aa7-8fb0-750e7e422b6b","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/d1746a6d05a30ca027012a7da4545962?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dkedwards%26avatarId%3D10200%26noRedirect%3Dtrue"},"displayName":"Ken Edwards","active":true,"timeZone":"America/Los_Angeles"},"created":"2013-06-27T16:19:15.281-0500","updated":"2013-06-27T16:19:15.281-0500"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/22078/comment/53072","id":"53072","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"body":"EUCA-7071 has additional info / steps to reproduce for this issue.","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"created":"2013-09-11T18:08:51.654-0500","updated":"2013-09-11T18:08:51.654-0500"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/22078/comment/53203","id":"53203","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"body":"The fix for this issue includes an upgrade task to remove the redundant \"vm_network_index\" column from the \"metadata_network_group\" table in the \"eucalyptus_cloud\" database.\n\nEUCA-7071 has details for reproducing the issue (on a setup using static network mode)","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=sjones","name":"sjones","key":"sjones","accountId":"557058:f5dce1b0-db45-4dd6-a0e8-3cea5effb5ce","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/037fddf6d3bf6d691a448e01136f233a?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dsjones%26avatarId%3D18300%26noRedirect%3Dtrue"},"displayName":"Steve Jones","active":true,"timeZone":"America/Los_Angeles"},"created":"2013-09-12T19:18:57.211-0500","updated":"2013-09-12T19:18:57.211-0500"},{"self":"https://eucalyptus.atlassian.net/rest/api/2/issue/22078/comment/53879","id":"53879","author":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=tony","name":"tony","key":"tony","accountId":"557058:353956db-b78c-4f3b-9096-f8f58e6709c8","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue"},"displayName":"Tony Beckham","active":true,"timeZone":"America/Tijuana"},"body":"Fix Verified.\r\n\r\nInstalled Packages\r\nName        : eucalyptus\r\nArch        : x86_64\r\nVersion     : 3.4.0\r\nRelease     : 0.0.1502.20130920git17d3b99d.el6\r\n\r\nThere is no longer vm_network_index column in metadata_network_group table. The error is not seen in the log after running example script from linked bug. Tested using system mode.\r\n\r\n{code}\r\neucalyptus_cloud=# select id,creation_timestamp,last_update_timestamp,version,metadata_network_group_id, metadata_display_name, metadata_account_name, vm_network_index from metadata_network_group;\r\nERROR:  column \"vm_network_index\" does not exist\r\nLINE 1: ...id, metadata_display_name, metadata_account_name, vm_network...\r\n{code}","updateAuthor":{"self":"https://eucalyptus.atlassian.net/rest/api/2/user?username=tony","name":"tony","key":"tony","accountId":"557058:353956db-b78c-4f3b-9096-f8f58e6709c8","avatarUrls":{"48x48":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=48&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3FownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","24x24":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=24&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","16x16":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=16&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dxsmall%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue","32x32":"https://avatar-cdn.atlassian.com/14ec1b18e630cc0743492fc327a12df7?s=32&d=https%3A%2F%2Feucalyptus.atlassian.net%2Fsecure%2Fuseravatar%3Fsize%3Dmedium%26ownerId%3Dtony%26avatarId%3D16600%26noRedirect%3Dtrue"},"displayName":"Tony Beckham","active":true,"timeZone":"America/Tijuana"},"created":"2013-09-23T20:00:11.161-0500","updated":"2013-09-23T20:00:11.161-0500"}],"maxResults":4,"total":4,"startAt":0},"customfield_11808":null,"worklog":{"startAt":0,"maxResults":20,"total":0,"worklogs":[]}}}